e2e/__tests__/__snapshots__/failures.test.ts.snap

```diff
@@ -228,7 +228,7 @@ FAIL __tests__/duringTests.test.js
       45 |
       46 | test('done(non-error)', done => {
 
-      at Object.<anonymous> (__tests__/duringTests.test.js:43:8)
+      at __tests__/duringTests.test.js:43:8
```

Comment:
I’m not exactly sure how to best handle this (and the below change). The main change in this PR (wrapping functions with a `done` callback in a named function) results in the user’s actual test function getting called in a different way between Jasmine and Circus, and thus in slightly different stack traces. These traces aren’t different in a meaningful way, but there’s no way to tell a snapshot test that.

Should I change the Circle runner, too, just so they match?

Reply:
Nevermind here — I realized this is really because I wasn’t preserving the test’s `this` value, which is important in Jest. Ensuring that made the stack traces the same again.